tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/VoteAPIHelper.java

```diff
@@ -230,7 +230,7 @@ private static VoteEntry[] calculateVotes(Set<User> votes, Map<User, Integer> ov
             if (overrides != null) {
                 override = overrides.get(u);
             }
-            entries.add(new VoteEntry(u.id, u.getLevel().getVotes(), override));
+            entries.add(new VoteEntry(u.id, u.getLevel().getVotes(u.getOrganization()), override));
```

Comment:
u.getLevel().getVotes(u.getOrganization())
could be shortened to
u.getVoteCount()